src/libraries/System.Formats.Tar/tests/TarReader/TarReader.Tests.cs

```diff
@@ -0,0 +1,99 @@
+using System.Xml.Linq;
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.IO;
+using Xunit;
+using System.Linq;
```

Comment:
```suggestion
// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license.

using System.Collections.Generic;
using System.IO;
using System.Linq;
using System.Xml.Linq;
using Xunit;
```

Reply:
Oh, VS Code added `using System.Xml.Linq;` automatically, not sure why. It's not being used.